src/PrestaShopBundle/Install/Install.php

```diff
@@ -99,6 +104,7 @@ public function __construct($settingsFile = null, $bootstrapFile = null)
 
         $this->settingsFile = $settingsFile;
         $this->bootstrapFile = $bootstrapFile;
+        $this->isDebug = _PS_MODE_DEV_;
```

Comment:
Why not putting it in the constructor like you did in [src/Adapter/Admin/PagePreference.php](https://github.com/PrestaShop/PrestaShop/pull/21515/files#diff-e7a4c7c6f16c191693fb9ca14dcd6296c769f04ea35f21183868da5cdc9d48c4R57)?

Reply:
Because you are in Install <3